lib/billy/browsers/watir.rb

```diff
@@ -0,0 +1,55 @@
+require 'billy'
+require 'watir-webdriver'
+
+module Billy
+  module Browsers
+    class Watir < ::Watir::Browser
+
+      def initialize(name, args = {})
+        @defaults = {
+          chrome: {
+            switches: %W[--proxy-server=#{Billy.proxy.host}:#{Billy.proxy.port}]
+          },
+          phantomjs: {
+            args: %W[--proxy=#{Billy.proxy.host}:#{Billy.proxy.port}]
+          },
+          firefox: {
+            profile: Selenium::WebDriver::Firefox::Profile.new,
+            proxy: Selenium::WebDriver::Proxy.new(
+              http: "#{Billy.proxy.host}:#{Billy.proxy.port}",
+              ssl: "#{Billy.proxy.host}:#{Billy.proxy.port}"
+            )
+          }
+        }
+
+        args = case name
+          when :chrome then configure_chrome(args)
+          when :phantomjs then configure_phantomjs(args)
+          else configure_firefox(args)
```

Comment:
Since `name` is a required param, I think we should explicitly check for `:firefox` here and the `else` should raise an error with invalid param.